CMake/itkCompilerChecks.cmake

```diff
@@ -18,8 +18,8 @@ endif ()
 
 # Minimum compiler version check: Microsoft C/C++
 if (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC" AND
-    CMAKE_CXX_COMPILER_VERSION VERSION_LESS 19.14)
-  message(FATAL_ERROR "Microsoft Visual Studio 2017 15.7 (MSVC 19.14) or later is required.")
+    CMAKE_CXX_COMPILER_VERSION VERSION_LESS 19.2)
```

Comment:
Thanks for this PR, @dzenanz 👍 Just wondering, is this (`19.2`) the correct version number to check? Does it stop CMake configuration on compilers before VS2019?

Reply:
Note that CMake VERSION_LESS does "component-wise integer version number comparison", not floating point (or fixed point) comparison: https://cmake.org/cmake/help/latest/command/if.html#version-less